src/utils/pause/mapScopes/index.js

```diff
@@ -65,19 +65,23 @@ export async function buildMappedScopes(
     return null;
   }
 
-  const generatedAstBindings = buildGeneratedBindingList(
-    scopes,
-    generatedAstScopes,
-    frame.this
-  );
-
   const originalRanges = await loadRangeMetadata(
     source,
     frame,
     originalAstScopes,
     sourceMaps
   );
 
+  if (originalRanges.every(range => range.columnEnd === Infinity)) {
```

Comment:
Just for my knowledge, why would a `columnEnd` be `Infinity`?

Reply:
that is what the source-map library includes

Reply:
We should check `columnStart === 0` too I think.

Reply:
@darkwing Mappings are basically line/col => line/col, so you might have a mapping saying "original line 4 column 2 to generated line 6 column 4". If there aren't any other mappings on original line 4, you'll end up with a mapping range saying "line 4 column 2 to column Infinity" because it doesn't have any other information about where the mapping locally might end.

Reply:
Thank you for the explanation @loganfsmyth !